QandA.md

```diff
@@ -3,51 +3,51 @@
 
 ### Was ist eine Nicht-24-Stunden-Schlaf-Wach-Störung?
 
-Die _Die Nicht-24-Stunden-Schlaf-Wach-Störung (Non-24)_ ist eine neurologische Schlafstörung in welcher der Schlaf-Wach-Rhythmus einer Person länger (oder in seltenen Fällen kürzer) als 24 Stunden ist. Die Person ist nicht in der Lage ihren Schlaf-Wach-Rhythmus an die Länge eines Tages anzupassen und ihre Schlafzeiten wandern stetig um die Uhr. Als Beispiel, geht die Person an einem Tag um Mitternacht ins Bett und schläft bis 8 Uhr früh, wird sie wahrscheinlich in der darauffolgenden Nacht nicht vor 1 oder 2 Uhr früh einschlafen können und deshalb auch am folgenden Morgen entsprechend länger schlafen müssen. Die Verspätung bleibt aber nicht um 1 oder 2 Uhr früh stehen, sondern wird jeden Tag schlimmer, bis sie um 4 Uhr, dann um 6 Uhr, 8 Uhr, 10 Uhr, mittags, 14 Uhr usw. ins Bett geht. Schlussendlich wandert sie um die Uhr bis zu dem Ausgangszeitpunkt und es geht wieder von vorne los. Bei einigen Personen verändert sich der Zeitraum von Tag zu Tag und sie können ihren Schlaf-Wach-Rhythmus nicht vorausberechnen.
```

Comment:
- Replaced double "die" (eng.: the) with single "die"
- Added missing commas
- Rearranged third sentence for better reading flow and grammatical accuracy